src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format(ttf),
+    url(./fonts/Avenir-Heavy.ttf) format(ttf);
+
+}
```

Comment:
Please, follow the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/99403198/226096028-5b315af8-4797-458f-b842-dc42df0a9b42.png">